src/GUI/MenuItem.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
```

Comment:
Should this declare be on line 2?

Not a blocking thing.
If you change, you could also add ending dots at line 13 and 14.

Reply:
I just grepped through the code for "strict_types" and it seems to be on the same line as `<?php` in all the other files in our code.

Reply:
Yes, I saw that in a few others files I reviewed this morning. Wasn't really sure about that.